README.md

```diff
@@ -1,16 +1,16 @@
-# Exchange Union
+# XUD
```

Comment:
The more I think about this the more I prefer "Exchange Union Daemon" for the title. Someone not familiar with the project probably won't be able to make sense of "XUD" right away.

Also, mostly a question here but should we have a preferred capitalization style? I usually see it as lower case "xud" which I think I prefer.